src/lib.rs

```diff
@@ -1542,7 +1542,27 @@ impl Bindings {
             }
         }
 
+        #[cfg(unix)]
+        fn can_read(perms: &std::fs::Permissions) -> bool {
+            use std::os::unix::fs::PermissionsExt;
+            perms.mode() & 0o400 > 0
```

Comment:
I'm not sure if this check is reliable enough...
The check assumes that bindgen is running under the user who owns this file which may not be the case.

Reply:
I think `perms.mode() & 0o444 > 0` (rather than `0o400`) is Good Enough(tm)